packages/networks/src/networks/arbitrum.ts

```diff
@@ -42,6 +42,9 @@ export const arbitrum: NetworkConfig = {
   isTestNetwork: false,
   description:
     'Arbitrum One is a Layer 2 (L2) chain running on top of Ethereum Mainnet that enables high-throughput, low cost smart contracts operations.',
+  uniswapV3: {
```

Comment:
I am not sure we need these for anything but mainnet?

Reply:
I added it because there was one. If someone else wants to use the uniswap service for other network. They can do it.

Reply:
fair point